ci/update.sh

```diff
@@ -23,5 +23,5 @@ rm -rf logs.txt
 nohup ${JAVA_HOME}/bin/java JVM_PARAMS \
     -Xdebug -Xrunjdwp:transport=dt_socket,server=y,address=DEBUG_PORT,suspend=n \
     -jar APP_PATH/APP_NAME \
-    --spring.config.location=classpath:/application.yml,APP_PATH/application.yml \
+    --spring.config.location=classpath:/application.yml,APP_PATH/ \
```

Comment:
this will break web application

Reply:
As explained in the docs, spring-boot will load application.yml from the given folder.